src/rulesets/rest/2022-05-25/json-api-rules/status-code-rules.ts

```diff
@@ -69,13 +73,12 @@ const post2xxCodes = new ResponseRule({
     response.statusCode.startsWith("2") &&
     rulesContext.operation.method === "post",
   rule: (responseAssertions) => {
-    const validPost2xxCodes = ["201"];
     responseAssertions.added(
       "support the correct 2xx status codes",
       (response) => {
         if (!validPost2xxCodes.includes(response.statusCode)) {
           throw new RuleError({
-            message: `expected POST response to only support 201, not ${response.statusCode}`,
+            message: `expected POST response to only support ${validPost2xxCodes[0]} or ${validPost2xxCodes[1]}, not ${response.statusCode}`,
```

Comment:
Let's not do that. Can we call `toString` on `validPost2xxcCode` instead?

Reply:
updated to use toString